packages/betterer/src/printer.ts

```diff
@@ -16,8 +17,10 @@ export function print(results: BettererResults): string {
           ? (value as Printable).print()
           : JSON.stringify(value);
     }
-    printedValue = (printedValue as string).replace(/`/g, '\\`');
-    return `\nexports[\`${resultName}\`] = {\n  timestamp: ${timestamp},\n  value: \`${printedValue}\`\n};`;
+    return `\nexports[\`${resultName}\`] = {\n  timestamp: ${timestamp},\n  value: \`${escape(
+      printedValue as string,
+      '"\n'
```

Comment:
3. Part of the api of `safe-string-literal` is to provide the ability to _not_ escape certain characters. This is a bit of a nicety to ensure the output stays legible for things we're mostly sure we've already escaped. In this case we're _skipping_ escaping new lines (`\n`) and double quotes (`"`). Now it's important to acknowledge that this requires that serialized string values are properly escaped if they contain quotes. 

🏁 End of the tour

Reply:
Yeah this is really good, thanks a lot!